account_financial_risk_ext/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    credit_balance = fields.Monetary(compute="_compute_risk_exception")
```

Comment:
Should we add `string="Credit Balance"`?

Reply:
@TimLai125 It is not necessary as the string automatically becomes "Credit Balance", although adding it makes it more explicit.